proto/buoyant-cloud-api.proto

```diff
@@ -125,6 +125,34 @@ message LinkerdMessage {
   }
 }
 
+//
+// ManageAgent Messages
+//
+
+message GetProxyDiagnostics {
+  string diagnostic_id = 1;
+  string pod_name = 2;
+  string pod_namespace = 3;
+}
+
+message AgentCommand {
+  oneof command {
+    GetProxyDiagnostics get_proxy_diagnostics = 1;
+  }
+}
```

Comment:
i like this approach 👍 . naming the `AgentCommand.GetProxyDiagnostics` to match the `ProxyDiagnostics` rpc satisfies my primary concern around making explicit which commands map to which messages/apis.

...re: @klingerf's comment:
> If we're worried about maintaining a more principled relationship between commands and responses, I suggest that we address that with consistent naming across our API definition. For example, we could provide a ProxyDiangostics endpoint that mirrors the naming of the GetProxyDiagnostics command that we're adding here. That said, I don't really mind that the current implementation is calling the LinkerdInfo endpoint instead.

adding `ProxyDiagnostic` to `LinkerdMessage` `oneof` would also satisfy my primary concern. i realize we built `LinkerdMessage` with a `oneof` to be extensible, but i have a slight preference for the separate `ProxyDiagnostics` api as the alternative feels like overloading `LinkerdInfo`. also implementing `ProxyDiagnostics` as a separate api makes the implementation clearer.

ultimately you're (@zaharidichev) the one who will spend the majority of the time implementing/testing/maintaining this (at least in the short term), so i'll defer to you on the approach here.